### PR TITLE
FILE-4 - move the node list from a char box in the upper right to a dedicated column on the left side of the page

### PR DESCRIPTION
**Pull request by gitstart**
### What does this PR do?

-  Move the node list from a char box in the upper right to a dedicated column on the left side of the page
-  2nd column, node id, remains, but the copy button is hidden by default and only becomes displayed, position: absolute on mouseover. thus it takes up no permanent, static space
-  4th column, payout status, can be shrunk by replacing valid and pending with two different icons that, on mouseover, show the full Valid or Pending. for example, a green checkbox logo that, on mouseover, displays Valid and a yellow clock logo that, on mouseover, displays Pending
- 6th column, retrievals, shorten from the full 72,102, 4,003,018 to use metric suffix: 72k, 4M, etc

### Issue reference
- https://github.com/filecoin-saturn/L1-dashboard/issues/38

### Demo video/scrennshot:
- https://www.loom.com/share/b5bbc2c74e5b4aceae96f419c952482f?sid=31746fe1-bdb1-4044-a914-4efb8e05f082

> This code was written and reviewed by GitStart Community. Growing future engineers, one PR at a time.

### CODE REVIEW

**Comment by gitstart** [comment]
Updated loom video after #52 got merged https://www.loom.com/share/0e1045977a364cf6abef80843dc4f771?sid=8d2d3906-2693-4322-b1cf-03b9d53d3a4d

**Comment by gitstart** [comment]
Good day @guanzo this PR is ready for review

**Comment by gitstart** [comment]
> Good day @guanzo this PR is ready for review

@guanzo just a gentle reminder that this PR is still up for review

**Comment by gitstart** [comment]
@DiegoRBaquero this PR is ready for review

**Comment by gitstart** [comment]
@guanzo @joaosa @DiegoRBaquero this PR is ready for review

**Review by DiegoRBaquero — APPROVED** [review]
lgtm but want a second approver

**Comment by guanzo** [comment]
Apologies for the late review.

Can you add an icon for the "Postponed" status? It'd be best if all statuses had an icon. Perhaps the same icon as the "In Progress" but with a gray color? Up to you.

**Comment by gitstart** [comment]
@guanzo updated
img1

**Review by joaosa — APPROVED** [review]
LGTM

**Pull request closed without merging**
